src/ign.cc

```diff
@@ -192,7 +192,6 @@ extern "C" IGNITION_GAZEBO_VISIBLE int runServer(const char *_sdfString,
       //   In Blueprint and Citadel, LogRecord will record to <path> if no
       //   --record-path is specified on command line.
       serverConfig.SetLogRecordPath(recordPathMod);
```

Comment:
This line can be moved outside of and after the if-else statement, and the TODO deleted.

Reply:
@mabelzhang , while you're looking at the empty `LogRecordPath`, mind moving this too? I'm not sure exactly what `if / else` it's referring to, and since all tests pass at the current state, I don't know if we should add a test to cover what will be changed by this move, or change an existing test's expectation.